dockerfiles/salsa/1.1.6/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:22.04
+
+# DO NOT EXPOSE ANY PORTS IN THIS FILE
+
+# Install necessary packages (Java and curl)
+RUN apt-get update \
+    && apt-get install -y openjdk-17-jdk curl --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /usr/src/app
+
+RUN mkdir -p /usr/local/lib/salsa
+
+# Download the Salsa compiler jar file
+RUN curl -SL -o /usr/local/lib/salsa/salsa1.1.6.jar http://wcl.cs.rpi.edu/salsa/releases/salsa1.1.6.jar
```

Comment:
Since this uses http, can you add a hash check on this. sha256 would be fine to use.